### PR TITLE
Solve edits modals

### PR DESCRIPTION
**Pull request by allthesignals**

### CODE REVIEW

File: app/models/organization.rb [hunk 1]
```diff
@@ -1,4 +1,6 @@
 class Organization < ActiveRecord::Base
+  extend Enumerize
```
Comment (beechnut) [inline app/models/organization.rb]: Why is `Enumerize` included?

File: db/schema.rb [hunk 2]
```diff
@@ -11,6 +11,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
+
```
Comment (beechnut) [inline db/schema.rb]: Would you delete the empty space at line 14?

File: db/migrate/20151231195856_add_type_to_organization.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class AddTypeToOrganization < ActiveRecord::Migration
+  def change
+    add_column :organizations, :type, :boolean
```
Comment (beechnut) [inline db/migrate/20151231195856_add_type_to_organization.rb]: This doesn't seem relevant to this PR.

File: app/models/organization.rb (hunk 1 above)
Comment (allthesignals) [inline app/models/organization.rb]: Because we talked about using Enumerize on Organization

**Comment by beechnut** [comment]
Closed via 9972fa7f0f9cf337
